chore(README): update features (#1231)
prisma/language-tools: packages/vscode/README.md

## Features

- Syntax highlighting of `schema.prisma`
- Symbols for schema blocks `datasource`, `generator`, `model`, `enum` and `type`

>   - They are listed in the `OUTLINE` view of VS Code and clicking them will go to their definition.

Reviewer: I think I would move this down below "Code Completion", and above "Documentation help" importance wise.

  - Diagnostic tools are used to surface errors and warnings in your schema file as you type.
- Code Completion
  - Completion results appear for symbols as you type.
  - You can trigger this manually with the <kbd>Ctrl</kbd>+<kbd>Space</kbd> shortcut.
- Symbols for schema blocks `datasource`, `generator`, `model`, `enum` and `type`

>   - They are listed in the `OUTLINE` view of VS Code and clicking them will go to their definition.  

Reviewer: ```suggestion
  - They are listed in the [Outline view of VS Code](https://code.visualstudio.com/docs/getstarted/userinterface#_outline-view) and clicking them will go to their definition.  
```

- Documentation help
  - Documentation of a completion result pops up as completion results are provided.
- Quick info on hover
  - Documentation Comments (`///`) of `model`, `enum` or `type` appear anywhere you hover over their usages.
  - CMD + hover on a field whose type is a `model`, `enum` or `type` will show the corresponding block in a popup.
- Go to Definition

>   - Jump to or peek a `model`, `enum` or `type` definition. (CMD + left click)

Reviewer: Does a readme file maybe support `<kbd>` to make <kbd>CMD</kbd> maybe?

Author: Nice
```suggestion
  - Jump to or peek a `model`, `enum` or `type` definition. (<kbd>CMD</kbd> + left click)
```

- Quick-fixes
  - Quickly fix typos in model and enum names
  - Create new models and enums with a single click

>   - For the "Error parsing attribute "@relation": The argument `references` must refer to a unique criteria in the related model. Consider adding an `@unique` attribute to the field `field` in the model `A`." it will add an `@unique` atribute where it's needed.

Reviewer: ```suggestion
  - Add `@unique` on model fields where it is missing for a `@relation`, `references` value
```

Author: The end of your sentence is a bit weird 🤔 I don't understand 
```
, `references` value in another model
```

Reviewer: It is the `references` value of a `@relation` that is pointing to a field that should have a `@unique`.

Author: So...
>Add `@unique` on model fields for a `@relation` where the `references` value is pointing to a field missing it.

Reviewer: Not the greatest English, but works for me. It is not the easiest problem to explain.

Reviewer: ```suggestion
  - Add `@unique` on model fields for a `@relation` where the `references` value is pointing to a field missing it.
```
